.github/workflows/ubuntu.yml

```diff
@@ -87,3 +86,79 @@ jobs:
             cat ${resultfile}
             exit 1
           fi
+
+  build_and_except_eula_manually:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Install brew packages
+        run: |
+          brew install cabextract
+          brew install fontforge
+
+      - name: Run script
+        run: |
+          yes "yes" | ./vista-fonts-installer.sh
+
+      - name: Test if directory is created with desired fonts
+        env:
+          MS_FONT_PATH: /usr/share/fonts/truetype/vista
+        run: |
+          testfile=$(pwd)/tests/expected-files-macos-raw.txt
+          resultfile=$(pwd)/result.txt
+          echo -n "" > ${resultfile}
+          pushd "$MS_FONT_PATH"
+          while IFS= read -r line
+          do
+            echo "$line"
+            if [ -f "$line" ]; then
+              echo "$MS_FONT_PATH/$line @ PRESENT" >> ${resultfile}
+            else
+              echo "$MS_FONT_PATH/$line @ MISSING" >> ${resultfile}
+            fi
+          done < "$testfile"
+          popd
+          if [ "0" == "$(grep -v MISSING ${resultfile} | wc -l | tr -d ' ')" ]; then
+            echo "At least one font file not being installed, see output:"
+            cat ${resultfile}
+            exit 1
+          fi
+
+  build_and_reject_eula_manually:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Install brew packages
+        run: |
+          brew install cabextract
+          brew install fontforge
+
+      - name: Run script
+        run: |
+          yes "no" | ./vista-fonts-installer.sh
+
+      - name: Test if directory is created with desired fonts
+        env:
+          MS_FONT_PATH: /usr/share/fonts/truetype/vista
+        run: |
```

Comment:
Can we move the test script to a separate script so that we can simplify the GHA workflow? Thanks.